GUIDE.md

```diff
@@ -1,50 +1,50 @@
-# Компоненты YARCH
+# YARCH components
 
 ## Builder
-Отвечает за сборку модуля. В нем создаются необходимые сущности и проставляются зависимости.
+Responsible for the assembly of the module. It creates the necessary entities and shows dependencies.
 
 ## DataFlow
-Представляет собой описание объектов для передачи данных (DTO - Data Transfer Objects) в рамках одного варианта использования (UseCase). Состоит из:
-* *Request* - структура для передачи данных из ViewController в Interactor
-* *Response* - структура для передачи данных из Interactor в Presenter
-* *ViewModel* - структура для передачи данных из Presenter во ViewController
-Структуры Request, Response, ViewModel, объявляются внутри enum с наименованием модуля для выделенного пространства имен. DataFlow должен четко описывать бизнес задачи и давать наглядную информацию о передаваемых данных внутри модуля.
+Description of objects for data transfer (DTO - Data Transfer Objects) within one use case (UseCase). Consists of:
+* * Request * - structure for transferring data from ViewController to Interactor
+* * Response * - structure for transferring data from Interactor to Presenter
+* * ViewModel * - structure for transferring data from Presenter to ViewController
+The Request, Response, ViewModel structures are declared inside the enum with the module name for the allocated namespace. DataFlow should clearly describe the business tasks and provide visual information about the data transmitted within the module.
 
 ## Interactor
-Содержит бизнес логику модуля. Нежелательно заставлять Interator ходить в сеть, обращаться к базе данных и т.д. Он выступает агрегатором бизнес логики. Так, например, для получения данных он обращается к Provider, к различным Worker'ам для выполнения каких-либо действий с данными, после чего передает подготовленные данные дальше в Presenter и т.д.
-Interactor получает запрос на операцию над данными от ViewController с помощью Request и предает данные в Presenter с помощью Response.
+Contains the business logic of the module. Interactor should not make network requests, access the database, etc. It acts as an aggregator of business logic. So, for example, to get data, it calls the Provider, to various Workers to perform any data actions, and then sends the prepared data further to the Presenter, etc.
+Interactor receives a request for an operation on data from the ViewController with the Request and submits the data to the Presenter using Response.
 
 ## Worker
-Реализует переиспользуемую часть бизнес логики (например: алгоритм фильтрация данных).
+Implements the reused part of the business logic (for example: algorithm for data filtering).
```

Comment:
Worker should implement business logic that can be reused in another module (for example: algorithm for data filtering).